test/src/e2e_vm_tests/test_programs/should_pass/static_analysis/cei_pattern_violation/Forc.toml

```diff
@@ -0,0 +1,6 @@
+[project]
+name = "cei_pattern_violation"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+implicit-std = true
```

Comment:
When `implicit-std` is set to `true` (which is also the default btw), the standard library is pulled from Github (from the latest release of `forc`), which we don't want in our tests. We always want to be using the version from `master` for compatibility reasons. So, you have two options:
* Set `implicit-std` to `false` if you _don't_ need the standard library in your test.
* Remove this field and add a path `std` dependency to where the standard library lives.